test/cypress/integration/widgets/blueprints_spec.ts

```diff
@@ -463,14 +463,22 @@ describe('Blueprints widget', () => {
             cy.contains('.item', source).click();
         }
 
+        function initializeFirstSegmentRow(segmentName: string) {
```

Comment:
How about:
```suggestion
        function addFirstSegmentRow(segmentName: string) {
```
The term 'initialize' is not very common in terms of UI controls.

Reply:
Sure, in that case I'll modify that accordingly 😉